### PR TITLE
Added additional files to github workflow

### PR DESCRIPTION
**Pull request by brettdavis4**

### CODE REVIEW

**Review by korymarks — APPROVED** [review]
LGTM

File: .github/workflows/code-checks.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      - name: "Setup Node"
+        uses: actions/setup-node@v1
+        with:
+          registry-url: "https://npm.pkg.github.com/"
+          scope: "@polarislabs"
+      - name: Switch to the Node version in the .nvmrc
+        shell: bash -l {0}
+        run: nvm install
+      - name: "Install dependencies"
+        run: npm install
```
Comment (babennettdev) [inline .github/workflows/code-checks.yml:40]: `npm install` or `npm ci`?

File: .github/workflows/publish-package.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - name: Use Node
+        uses: actions/setup-node@v1
+        with:
+          registry-url: https://npm.pkg.github.com
+          scope: "@polarislabs"
+      - name: Switch to the Node version in the .nvmrc
+        shell: bash -l {0}
+        run: nvm install
+      - name: Install dependencies
+        run: npm install
```
Comment (babennettdev) [inline .github/workflows/publish-package.yml:21]: `npm install` or `npm ci`?

**Review by babennettdev — CHANGES_REQUESTED** [review]
Brett, these look good, but I'm curious if we should use `npm install` or `npm ci` for installing dependencies. Thoughts?

File: .github/workflows/code-checks.yml (hunk 1 above)
Comment (brettdavis4) [reply .github/workflows/code-checks.yml]: @dbirks and I talked about this and it's been changed to npm ci.

File: .github/workflows/publish-package.yml (hunk 2 above)
Comment (brettdavis4) [reply .github/workflows/publish-package.yml]: It's been changed to npm ci.

File: .github/workflows/code-checks.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+name: Code Checks
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (dbirks) [inline .github/workflows/code-checks.yml:6]: I think we can run the code checks on PR's to develop as well.

**Review by dbirks — CHANGES_REQUESTED** [review]
Cool, I could only find one small thing, thanks!

File: .github/workflows/code-checks.yml (hunk 3 above)
Comment (brettdavis4) [reply .github/workflows/code-checks.yml:6]: I've added develop to the branches list.

Comment (dbirks) [reply .github/workflows/code-checks.yml:6]: Nice, thanks :+1:

**Review by dbirks — APPROVED** [review]

File: .github/workflows/code-checks.yml (hunk 1 above)
Comment (brettdavis4) [reply .github/workflows/code-checks.yml]: I've made the change.  Can you please approve it?  thanks!

**Review by babennettdev — APPROVED** [review]
Thanks for updating to `npm ci` 👍 Looks good.

**Pull request merged**